.github/workflows/validation.yml

```diff
@@ -41,6 +41,8 @@ jobs:
         run: yarn dedupe --check
       - name: install deps
         run: yarn --immutable
+      - name: Check commit messages
```

Comment:
Can you create separate GH flow for this? It will only block whole validation before if you don't rebase before, which is something we don't want. Also there is already too much stuff in validation.yml and I plan to move more things to separated flows.

Also you don't need to install deps because of this scripts so it should be super fast flow.

Reply:
You don't need to involve yarn/node at all just call that shell script directly

Reply:
Got it! ccd5d47f64e20c208ea599c507d3bd83424c319d